mlrun/db/base.py

```diff
@@ -666,6 +666,7 @@ def create_user_secrets(
     def create_model_endpoint(
         self,
         model_endpoint: mlrun.common.schemas.ModelEndpoint,
+        creation_strategy: mlrun.common.schemas.ModelEndpointCreationStrategy = "inplace",
```

Comment:
Add `import mlrun.common.schemas.model_monitoring.constants as mm_constants` and then use it (with no `,` in the end)
`creation_strategy: mm_constants.ModelEndpointCreationStrategy =mm_constants.ModelEndpointCreationStrategy.INPLACE`
```suggestion
        creation_strategy: mlrun.common.schemas.ModelEndpointCreationStrategy = "inplace"
```